core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/StreamUtil.java

```diff
@@ -165,6 +168,43 @@ public static InputStream opAuths(final Class clazz, final boolean logErrors) {
         return schemas;
     }
 
+
+    public static InputStream[] openStream(final String... urls) throws IOException {
```

Comment:
Perhaps remove these methods and just have the methods that take a URL. This method could be confused with the open stream from path.